pkg/build/builder/docker.go

```diff
@@ -12,7 +12,6 @@ import (
 	dockercmd "github.com/docker/docker/builder/command"
 	"github.com/docker/docker/builder/parser"
 	docker "github.com/fsouza/go-dockerclient"
-	"github.com/golang/glog"
```

Comment:
It's mostly transitional, to avoid me forcing you to refactor all this code
right now.

On Sat, May 7, 2016 at 7:54 PM, Ben Parees <notifications@github.com> wrote:

> In pkg/build/builder/docker.go
> <https://github.com/openshift/origin/pull/8786#discussion_r62424375>:
>
> > @@ -12,7 +12,6 @@ import (
> >  	dockercmd "github.com/docker/docker/builder/command"
> >  	"github.com/docker/docker/builder/parser"
> >  	docker "github.com/fsouza/go-dockerclient"
> > -	"github.com/golang/glog"
>
> i guess i can appreciate keeping the name the same so people writing code
> don't have to think about it, but i also dislike the idea that people are
> using it w/o realizing it's different.
> [shrug].
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8786/files/9725d5171cdd040a8f619a1a7d7b332d9228209b#r62424375>
>